src/oc/storage/resources/board.clj

```diff
@@ -53,28 +50,29 @@
   [board]
   (apply dissoc board ignored-properties))
 
-(schema/defn ^:always-validate drafts-storyboard :- common/Board
-  "Return a storyboard for the specified org and author."
-  [org-uuid :- lib-schema/UniqueID user :- lib-schema/User]
-  (let [now (db-common/current-timestamp)]
-    (merge default-drafts-storyboard {
-      :org-uuid org-uuid
-      :author (lib-schema/author-for-user user)
-      :authors [(:user-id user)]
-      :created-at now
-      :updated-at now})))
+;; REMOVE Once it's confirmed if drafts will have a board
+;; (schema/defn ^:always-validate drafts-storyboard :- common/Board
+;;   "Return a storyboard for the specified org and author."
+;;   [org-uuid :- lib-schema/UniqueID user :- lib-schema/User]
+;;   (let [now (db-common/current-timestamp)]
+;;     (merge default-drafts-storyboard {
+;;       :org-uuid org-uuid
+;;       :author (lib-schema/author-for-user user)
+;;       :authors [(:user-id user)]
+;;       :created-at now
+;;       :updated-at now})))
 
 ;; ----- Board Slug -----
 
-(def reserved-slugs #{"create-board" "settings" "boards" "stories" "journals" "all-activity" "activity"})
+(def reserved-slugs #{"create-board" "settings" "boards" "all-activity"})
```

Comment:
This should use `all-posts` now instead of `all-activity`

Reply:
Updated @bago2k4  Good catch.

Reply:
Reminded me we also need to reserve `drafts` here.